webextensions/api/tabs.json

```
@@ -2809,7 +2806,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤61"
+                    "version_added": "47"
```

:x: https://github.com/chromium/chromium/commit/09a3ccc44eb5b7552d6ec49dc8fb46678dd4025f in [45.0.2453.0](https://github.com/chromium/chromium/releases/tag/45.0.2453.0) before 45.0.2454.

```suggestion
                    "version_added": "45"
```